Sources/Quick/QuickSpec.swift

```diff
@@ -0,0 +1,29 @@
+import XCTest
+
+// NOTE: This file is not intended to be included in the Xcode project. It
+//       is picked up by the Swift Package Manager during its build process.
+
+public class QuickSpec: XCTestCase, XCTestCaseProvider {
+    public func spec() {}
+
+    public required init() {}
+
+    public var allTests : [(String, () -> Void)] {
```

Comment:
I'm sure you're following along with https://github.com/apple/swift-corelibs-xctest/commit/e5c0cb36bc445d0522f7ec28a8342b2681f68666, so you're probably also already aware these should be converted to `[(String, () throws -> Void)]`. Still, just making a note of it here. There's also a spot below that needs to be modified as well.

> Aside: Boy, how are we ever going to version this thing? :grin: